python/README.md

```diff
@@ -141,10 +169,42 @@ slos = ibmos2spark.softlayer(sc, configuration_name, auth_url, tenant, username,
 data = sc.textFile(slos.url(container_name, object_name))
 ```
 
+### Object Storage OpenStack Swift for Bluemix
+
+This is the service described in **far right** pane in the image above. 
+This was previously referred to as Bluemix Swift Object Storage and is still the default object
+storage service provided in Data Science Experience, as of 12 Oct 2017. Credentials are passed as 
```

Comment:
I think there is no point of mentioning what is default on DSX.

Reply:
Yeah, you're right that this isn't the place for it.